src/Faker/Provider/en_US/Payment.php

```diff
@@ -29,10 +29,126 @@ public function bankRoutingNumber()
 
     public static function calculateRoutingNumberChecksum($routing)
     {
-        return (
```

Comment:
Can you revert this unrelated change here, please?

Reply:
no problem, i'll do asap

Reply:
@xfudox bump